google-cloud-clients/google-cloud-os-login/src/test/java/com/google/cloud/oslogin/v1/it/ITSystemTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.oslogin.v1.it;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.oslogin.common.OsLoginProto;
+import com.google.cloud.oslogin.v1.FingerprintName;
+import com.google.cloud.oslogin.v1.ImportSshPublicKeyResponse;
+import com.google.cloud.oslogin.v1.LoginProfile;
+import com.google.cloud.oslogin.v1.OsLoginServiceClient;
+import com.google.cloud.oslogin.v1.UserName;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ITSystemTest {
+  private static OsLoginServiceClient osLoginServiceClient;
```

Comment:
Is it necessary to keep this client as a static field? Can you please describe what is the use case behind it?

Reply:
yes its necessary to keep this client as a static field because `beforeClass()` and `afterClass()` method always be static.

Reply:
You can always use @Before and @After if you just want to initialize and close the client

Reply:
@before and @after I tried it before when i wrote system test for redis but jeff suggest me to used  `@Beforeclass `and `@AfterClass`

Reply:
> You can always use @before and @after if you just want to initialize and close the client. I dont really understand how beforeClass() & afterClass() has to do with client being static?

Done